tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/LabelHelper.cs

```diff
@@ -57,6 +58,34 @@ public static ServiceLabelStatus CheckServiceLabel(string csvContent, string ser
             return ServiceLabelStatus.DoesNotExist;
         }
 
+        public static bool CheckServiceLabelInReview(IReadOnlyList<Octokit.PullRequest?> pullRequests, string serviceLabel)
+        {
+            try
+            {
+                if (pullRequests == null || !pullRequests.Any())
+                {
+                    return false;
+                }
+
+                foreach (var pr in pullRequests.Where(p => p != null))
+                {
+                    if (
+                    pr != null &&
+                    pr.Title.Contains(serviceLabel, StringComparison.OrdinalIgnoreCase) &&
+                    pr.Labels.Any(label => label.Name.Equals("Created by copilot")))
+                    {
+                        return true;
+                    }
+                }
+
+                return false;
+            }
+            catch (Exception)
+            {
+                return false;
```

Comment:
We should not catch exceptions here, it could swallow actual bugs (and not log them!) and then return false negatives. What would be an expected exception? I assume only a potential null error, but we should just handle that properly in the code.